test/DebugInfo/NonSemantic/DebugFunction.cl

```diff
@@ -4,7 +4,7 @@
 // - Parent operand of DebugFunction is DebugCompilationUnit, not an OpString,
 //   even if in LLVM IR it points to a DIFile instead of DICompileUnit.
 
-// RUN: %clang_cc1 %s -cl-std=clc++ -emit-llvm-bc -triple spir -debug-info-kind=line-tables-only -O0 -o - | llvm-spirv -o %t.spv
+// RUN: %clang_cc1 %s -cl-std=clc++ -emit-llvm-bc -triple spir -debug-info-kind=line-tables-only -O0 -o - | llvm-spirv --spirv-debug-info-version=nonsemantic-shader-100 -o %t.spv
```

Comment:
Please add new RUN line with --spirv-debug-info-version=nonsemantic-shader-200

Reply:
ah, it's my bad that I didn't add it earlier in the first place. anyway, please add it :)